src/routes/_actions/edit.js

```diff
@@ -15,9 +15,11 @@ export async function edit (status) {
     if (replyingTo) inReplyToHandle = '@' + replyingTo.account.acct
   }
   store.clearComposeData('dialog')
+
+  const pleromaAkkomaSpecificAPI = source.pleroma || source.akkoma
```

Comment:
this won't work because akkoma also sends source.pleroma, but it only sends MFM in source.akkoma so that needs to be first